src/shared/components/Orgs/OrgList.tsx

```diff
@@ -1,83 +1,162 @@
 import * as React from 'react';
-import { Input, Empty } from 'antd';
-import OrgItem, { OrgItemProps } from './OrgItem';
-import AnimatedList from '../Animations/AnimatedList';
-
+import { Empty, Pagination, Spin, Button } from 'antd';
+import { OrgResponseCommon, OrganizationList } from '@bbp/nexus-sdk';
+import { PaginationSettings } from '../../utils/types';
+import Search from 'antd/lib/input/Search';
+import OrgItem from './OrgItem';
+import { AccessControl, useNexus } from '@bbp/react-nexus';
 import './Orgs.less';
-import { Organization } from '@bbp/nexus-sdk-legacy';
 
-export interface OrgListProps {
-  orgs: OrgItemProps[];
-  busy?: boolean;
-  error?: { message: string; name: string };
-  onOrgClick?(org: Organization): void;
-  onOrgEdit?(label: string): void;
-  paginationSettings?: { total: number; from: number; pageSize: number };
-  onPaginationChange?: (page: number, pageSize?: number) => void;
+export interface OrgListContainerProps {
+  pageSize?: number;
+  onOrgClick?(orgLabel: string): void;
+  onOrgEdit?(orgLabel: string): void;
+  createOrg?(): void;
 }
 
-const Search = Input.Search;
+export const OrgsListContainer: React.FunctionComponent<
+  OrgListContainerProps
+> = props => {
+  const { pageSize = 20, onOrgClick, onOrgEdit, createOrg } = props;
```

Comment:
We need to use the same default value everywhere (from a exported const)

Reply:
I used the default UI config state const, but I also broke it into a new file in `utils/consts` where I hope we can find these things easier